go/vumitools/api.py

```
@@ -224,6 +230,68 @@ def msg_options(self, tag, tagpool_metadata=None):
                                                    self.user_account_key)
         returnValue(msg_options)
 
+    @Manager.calls_manager
+    def acquire_tag(self, pool):
+        """Acquire a tag from a given tag pool.
+
+        Tags should be held for the duration of a conversation.
+
+        :type pool: str
+        :param pool:
+            name of the pool to retrieve tags from.
+        :rtype:
+            The tag acquired or None if no tag was available.
+        """
+        user_account = yield self.get_user_account()
+        # TODO: Check that account has access to pool.
```

My vote would be to implement it here, since this is the `VumiUserApi` it should raise an error if it tries to do something it's not allowed to do.